src/settings/helpers/FeatureSettingWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
This file should probably be in the components directory rather than here. Somewhere alongside the existing setting components (`LabelledSettingToggle`, from memory) would work

Reply:
Putting it into /components/ makes it skinnable (via getComponent which feels kinda redundant, it shouldn't get bundled into the monolithic bundle because of that.